ci/verify_bundle.py

```diff
@@ -51,13 +51,17 @@ def _get_weights_names(bundle: str):
     # TODO: this function is temporarily used. It should be replaced by detailed config tests.
     if bundle == "brats_mri_generative_diffusion":
         return "model_autoencoder.pt", "model_autoencoder.ts"
+    if bundle == "brats_mri_axial_slices_generative_diffusion":
+        return "model_autoencoder.pt"
```

Comment:
please use:
```
return "model_autoencoder.pt", None
```
The output should have the same format (here: a tuple with length 2)

Reply:
Yes! Thank you!